index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>niicode</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <nav>
+        <h3 class="logo-text font-style">My Logo</h3>
+        <img class="menu-icon" src="/assets/images/Enabled.png" alt="blue menu icone">
+    </nav>
```

Comment:
Great usage of Semantic tags👏🏽
- [ ] It would be useful  to make the toolbar container a `header` to act as the root container of all toolbar menus  plus the logo  in order to  improve  more on the Semantics and website SEO